ax3_email/utils.py

```diff
@@ -116,7 +116,7 @@ def send_email(
 
 def _email_message_simple(subject, body, mail_to, reply_to, bcc, from_email):
     email_message = EmailMessage(
-        subject=subject,
+        subject=EMAIL_SUBJECT.format(subject),
```

Comment:
Revisar si se puede llamar directamente utilizando 
`from django.conf import settings`

```suggestion
        subject=settings.EMAIL_SUBJECT.format(subject),
```